scss/components/_switch.scss

```diff
@@ -208,6 +208,10 @@ $switch-paddle-transition: all 0.25s ease-out !default;
   // <input> element
   .switch-input {
     @include switch-input;
+      // indicate a disabled switch
+    &:disabled ~ label {
+      cursor: not-allowed;
```

Comment:
You can use `$switch-cursor-disabled` instead of a hard-coded property (many people don't like the `not-allowed` cursor.

Also, you could add `$switch-opacity-disabled` (with `.5` or `.66`) as default so we can recognize a disabled switch without having to move the cursor on it.